components/nimbus/src/dbcache.rs

```diff
@@ -98,9 +110,20 @@ impl DatabaseCache {
     }
 
     pub fn get_experiment_branch(&self, id: &str) -> Result<Option<String>> {
```

Comment:
Similarly, I'm guessing this wants to be get_enrolled_experiment_branch.  I also suspect we don't really want to make that change now since it would be a breaking change, but at least some comments could be helpful, and maybe a spinoff ticket if we think the method is likely to survive long enough to be worth renaming.

Reply:
Yes, you're right: this would be a breaking change :( Having said that, I think this method will becoming less important, though likely not going away.